src/libs/actions/Report.ts

```diff
@@ -1597,7 +1598,15 @@ function readNewestAction(reportID: string | undefined, shouldResetUnreadMarker
         lastReadTime,
     };
 
-    API.write(WRITE_COMMANDS.READ_NEWEST_ACTION, parameters, {optimisticData});
+    API.write(
+        WRITE_COMMANDS.READ_NEWEST_ACTION,
+        parameters,
+        {optimisticData},
+        {
+            checkAndFixConflictingRequest: (persistedRequests) => resolveReadNewestActionConflicts(persistedRequests, parameters),
```

Comment:
```suggestion
            checkAndFixConflictingRequest: (persistedRequests) => resolveDuplicationConflictAction(persistedRequests, (request) => request.command === WRITE_COMMANDS.READ_NEWEST_ACTION),
```

Isn't the latest call has the latest timestamp in the request queue?

https://github.com/Expensify/App/blob/36a723f1f4abd33752cfd6740e626c1d37bf9573/src/libs/actions/Report.ts#L1578-L1583

If yes then we can reuse the existing util function `resolveDuplicationConflictAction`

Reply:
Yeah, good point. I simplified `resolveReadNewestActionConflicts`

Reply:
Thanks @martasudol 

How about putting the request matcher here only to avoid an extra function(`resolveReadNewestActionConflicts`) creation?

```js
{
    checkAndFixConflictingRequest: (persistedRequests) =>
        resolveDuplicationConflictAction(persistedRequests, (request) => request.command === WRITE_COMMANDS.READ_NEWEST_ACTION && request.data?.reportID === parameters.reportID),
},
```